docs/API/NuGet-Protocols.md

```diff
@@ -1,89 +1,99 @@
+To interact with nuget.org, clients need to follow certain protocols. Because these protocols keep evolving, clients
+must identify the protocol version they use when calling specific nuget.org APIs. This allows nuget.org to introduce
+changes in a non-breaking way for the old clients.
 
-This topic lists various protocols as and when they come to existence.
+These APIs are specific to nuget.org and there is no expectation for other NuGet server implementations to introduce
+these APIs.
 
 ## NuGet protocol version 4.1.0
 
-This protocol specifies usage of verify-scope keys to interact with services other than nuget.org, to validate a package against a nuget.org account. 
+Th3 4.1.0 protocol specifies usage of verify-scope keys to interact with services other than nuget.org, to validate a
+package against a nuget.org account. 
 
-Validation ensures that the user-created API keys are used only with nuget.org, and that other verification or validation from a third-party service is handled through a one-time use verify-scope keys. These verify-scope keys can be used to validate that the package belongs to a particular user (account) on nuget.org.
+Validation ensures that the user-created API keys are used only with nuget.org, and that other verification or
+validation from a third-party service is handled through a one-time use verify-scope keys. These verify-scope keys can
+be used to validate that the package belongs to a particular user (account) on nuget.org.
 
 ### Client requirement
 
-If a client interacts with other services and needs to validate whether a package belongs to a particular user (account), it should use this protocol and use the verify scope keys and not the API keys from nuget.org.
-
 Clients are required to pass the following header when they make API calls to **push** packages to nuget.org:
 
 ```
 X-NuGet-Protocol-Version: 4.1.0
 ```
 
-### Protocol usage details
+Note that the pre-existing `X-NuGet-Client-Version` header has the same purpose but is now deprecated and should no
+longer be used.
+
+The **push** protocol itself is described in the documentation for the
+[`PackagePublish` resource](v3/package-publish-resource.md).
+
+If a client interacts with externals services and needs to validate whether a package belongs to a particular user
+(account), it should use the following protocol and use the verify-scope keys and not the API keys from nuget.org.
+
+### API to request a verify-scope key
 
-#### API to request a verify scope key
 This API is used to get a verify-scope key for a nuget.org author to validate a package owned by him/her.
 
 ```
-POST api/v2/package/create-verification-key/{id}/{version}
+POST api/v2/package/create-verification-key/{ID}/{VERSION}
 ```
 
-Request parameters:
+#### Request parameters
 
-| Name | In | Type | Notes|
-|:------------- |:-------------|:----|:-----|
-| id    | URL | string | Required: the package identidier for which theverify scope key is requested. |
```

Comment:
Suggest making a Required/Optional column instead of having it as the first part of the description.

Reply:
Fixed.